src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -884,6 +892,10 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
             ntracerio2d=ntracerio2d+1
           else if(trim(vartem)=='z') then
              write(6,*)'the metvarname ',trim(vartem),' will be dealt separately'
+          else if(trim(vartem)=='t2m') then
+             !write(6,*)'the metvarname ',trim(vartem),' will be dealt separately'
+          else if(trim(vartem)=='q2m') then
+             !write(6,*)'the metvarname ',trim(vartem),' will be dealt separately'
```

Comment:
Should the writes be committed out?  If they won't be used, please remove commented out lines.

Reply:
the lines are removed